hr_employee_data_from_work_address/init_hook.py

```diff
@@ -80,21 +55,50 @@ def adjust_employee_partners_post(env):
     if not employees.ids:
         return
     env.cr.execute(
-        'select work_phone, work_email, mobile_phone, image '
+        'select work_phone, work_email, mobile_phone '
         'from hr_employee where id in %s order by id',
         (tuple(employees.ids),))
     employee_db_data = env.cr.dictfetchall()
     for employee, db_data in zip(employees, employee_db_data):
         employee.address_id = env['res.partner'].create({
             'employee': True,
-            'name': employee.name,
+            'name': employee.name or employee.display_name,
```

Comment:
I think name of 'res.partner' should be updated by name employee, not display_name